### PR TITLE
Add perms for /opt/stackstorm/packs

### PR DESCRIPTION
**Pull request by DoriftoShoes**

### CODE REVIEW

**Comment by manasdk** [comment]
This happens after packs are installed?

File: modules/profile/manifests/st2server.pp [hunk 1]
```diff
@@ -222,6 +222,14 @@
     groups  +> $_st2_packs_group,
   }
 
+  file { '/opt/stackstorm/packs':
+    ensure  => 'directory',
+    owner   => 'root',
+    group   => $_st2_packs_group,
+    mode    => 0775,
```
Comment (manasdk) [inline modules/profile/manifests/st2server.pp]: Error: Parameter mode failed on File[/opt/stackstorm/packs]: The file mode specification must be a string, not 'Fixnum' at /opt/puppet/environments/packs_perms/modules/profile/manifests/st2server.pp:225

Comment (DoriftoShoes) [inline modules/profile/manifests/st2server.pp]: I'm going to move this in to puppet-st2.

**Comment by manasdk** [comment]
Confirmed that icons are all good and workflows are editable. Merge?

**Comment by DoriftoShoes** [comment]
Lets merge this to unblock the icons issue.  I am trying to figure out the best way to move this to puppet-st2 but I don't want to hold on that.

**Comment by manasdk** [comment]
Merging.

**Pull request merged**